docs/api/scenario-basics.md

```diff
@@ -0,0 +1,356 @@
+featured scenarios that were created or commissioned by professional and governmental bodies.
+Any of these scenarios can be used as a preset.
+
+When using a preset, all attributes and user values as described above are copied to a new scenario.
+All attached curves and heat- and flexibility orders are also cloned from the preset to the new scenario.
+The cloning is done only when the scenario using the preset is created. This means that later changes
+to the preset will not be reflected in the new scenario.
+
+Next to the given list of available presets by the ETM, you may also use your own scenarios as a preset.
+Also see the [create section](#create-a-scenario-based-on-another-scenario).
+
+## Get information about an existing Scenario
+
+Fetches all the attributes of the given scenario.
+
+<ApiEndpoint data={endpointData.show} />
+
+```http title="Example request"
+GET /api/v3/scenarios/12345 HTTP/2
+Host: engine.energytransitionmodel.com
+Accept: application/json
+```
+
+```http title="Example response"
+HTTP/2 200 OK
+Content-Type: application/json; charset=utf-8
+
+{
+  "id": 12345,
+  "title": "API",
+  "area_code":"nl",
+  "start_year": 2019,
+  "end_year": 2050,
+  "url": "https://engine.energytransitionmodel.com/api/v3/scenarios/12345",
+  "ordering": null,
+  "display_group": null,
+  "scaling": null,
+  "source": null,
+  "template": null,
+  "created_at": "2021-07-16T09:23:00.000Z",
+  "updated_at": "2021-07-16T09:23:00.000Z",
+  "protected": false,
+  "esdl_exportable": null
+}
+```
+
+### The `detailed` parameter
+
+The information supplied by the GET request will by default not contain slider settings. Adding
+the paramater `detailed` with a truthy value to the request will have the response include an extra
+object called `user_values`. This object contains a key, value pair for each slider that has been touched.
+
+```http title="Example request"
+GET /api/v3/scenarios/12345?detailed=true HTTP/2
+Host: engine.energytransitionmodel.com
+Accept: application/json
+```
+
+```http title="Example response"
+HTTP/2 200 OK
+Content-Type: application/json; charset=utf-8
+
+{
+  "id": 12345,
+  "title": "API",
+  "area_code": "nl",
+  "start_year": 2019,
+  "end_year": 2050,
+  ...
+  "user_values": {
+    "buildings_insulation_level": 40.3,
+    "capacity_of_energy_power_hydro_river": 39.0
+  }
+}
+```
+
+## Create a Scenario
+
+A scenario can be created in three ways: as a blank scenario for an area and end year, as a new scenario
+with certain sliders set, or as a scenario based on another scenario (its preset). All creation
+methods use a POST request on the scenario endpoint, but with different data.
+
+### Create a blank Scenario
+
+Creates a scenario where all sliders are at their default position. The following data can
+be supplied:
+
+* `area_code` - the code of the desired area, default: 'nl'
+* `title` - title of the scenario, default: 'API'
+* `end_year` - end year of the scenario, integer, default: 2015
```

Comment:
[The default end year is 2050](https://github.com/quintel/etengine/blob/9a4deb26af4dfbecc8efae51b74bd6678ae03b66/app/models/scenario.rb#L96).

```suggestion
* `end_year` - end year of the scenario, integer, default: 2050
```